picard/ui/coverartbox.py

```diff
@@ -357,20 +364,23 @@ def fetch_remote_image(self, url, fallback_data=None):
         if self.item is None:
             return
 
-        if fallback_data is not None:
+        if fallback_data:
             self.load_remote_image(url, None, fallback_data)
 
         if url.scheme() in ('http', 'https'):
-            path = url.path(QtCore.QUrl.FullyEncoded)
+            path = url.path()
             if url.hasQuery():
-                path += '?' + url.query(QtCore.QUrl.FullyEncoded)
+                query = QtCore.QUrlQuery(url.query())
+                queryargs = {k:v for k, v in query.queryItems()}
```

Comment:
I know this is my code and a bit nit-picking, but I think `k: v` instead of `k:v` reads nicer. In [PEP 274](https://www.python.org/dev/peps/pep-0274/), which introduced dict comprehension, they even use `k : v` (but not consistently).

Reply:
I agree for the style, but here the whole dict comprehension is useless, fixed in 
dac7c2